crates/bevy_animation/src/graph.rs

```diff
@@ -313,6 +420,57 @@ impl AnimationGraph {
         let mut ron_serializer = ron::ser::Serializer::new(writer, None)?;
         Ok(self.serialize(&mut ron_serializer)?)
     }
+
+    /// Adds an animation target (bone) to the mask group with the given ID.
+    ///
+    /// Calling this method multiple times with the same animation target but
+    /// different mask groups will result in that target being added to all of
+    /// the specified groups.
+    pub fn add_target_to_mask_group(&mut self, target: AnimationTargetId, mask_group: u32) {
+        *self.mask_groups.entry(target).or_default() |= 1 << mask_group;
+    }
+}
+
+impl AnimationGraphNode {
+    /// Masks out the mask groups specified by the given `mask` bitfield.
+    ///
+    /// A 1 in bit position N causes this function to mask out mask group N, and
+    /// thus neither this node nor its descendants will animate any animation
+    /// targets that belong to group N.
+    pub fn add_mask(&mut self, mask: AnimationMask) -> &mut Self {
+        self.mask |= mask;
+        self
+    }
+
+    /// Unmasks the mask groups specified by the given `mask` bitfield.
+    ///
+    /// A 1 in bit position N causes this function to unmask mask group N, and
+    /// thus this node and its descendants will be allowed to animate animation
+    /// targets that belong to group N, unless another mask masks those targets
+    /// out.
+    pub fn remove_mask(&mut self, mask: AnimationMask) -> &mut Self {
+        self.mask &= !mask;
+        self
+    }
+
+    /// Masks out the single mask group specified by `group`.
+    ///
+    /// After calling this function, neither this node nor its descendants will
+    /// animate any animation targets that belong to the given `group`.
+    pub fn add_mask_group(&mut self, group: u32) -> &mut Self {
+        self.mask |= 1 << group;
+        self
+    }
+
+    /// Unmasks the single mask group specified by `group`.
+    ///
+    /// After calling this function, this node and its descendants will be
+    /// allowed to animate animation targets that belong to the given `group`,
+    /// unless another mask masks those targets out.
+    pub fn remove_mask_group(&mut self, group: u32) -> &mut Self {
+        self.mask &= !(1 << group);
+        self
```

Comment:
```suggestion
        self.remove_mask(1 << group)
```

Nit: Same justification as above.